hazelcast/src/test/java/com/hazelcast/datastore/impl/ExternalDataStoreServiceImplTest.java

```diff
@@ -127,7 +136,7 @@ public void should_fail_when_datastore_class_NON_existing() throws Exception {
                 .setClassName("com.example.NonExistingClass");
         wrongConfig.addExternalDataStoreConfig(externalDataStoreConfig);
 
-        assertThatThrownBy(() -> createHazelcastInstance(wrongConfig))
```

Comment:
Minor : 
should_fail_when_datastore_class_NON_existing() and
should_fail_when_datastore_class_DOES_NOT_implements_ExternalDataStoreFactory()
methods have unnecessary "throws Exception" in the method signature

Reply:
fixed: https://github.com/hazelcast/hazelcast/pull/22678/commits/a544a9a22e0585a21b10cc6782d34d24a03f491c